MOTIVATION.md

```diff
@@ -0,0 +1,138 @@
+# Motivation
+
+The Human Cell Atlas (HCA) Data Coordination Platform (DCP) ingests,
+processes, and stores single-cell data with the goal of making that data easily
+available to a wide audience of researchers.
+The single-cell data type of greatest interest to most researchers is expression
+matrices, matrices that contain cell-by-gene expression values that are the
```

Comment:
I would simplify to _expression matrices ~~,~~ ~~matrices~~ that contain ..._

Reply:
Okay, I've shortened and removed the repetition.